common/environment/environment.hpp

```diff
@@ -66,7 +66,13 @@ enum EnvMachineOS
     MachineOsSize = 4
 };
 
-
+enum RoxieTargetType
+{
+    RTTUnknown = 0,
+    RTTQueriesOnly = 1,
+    RTTNotQueriesOnly = 2,
```

Comment:
Since you have three options, I think the names would be simpler as something like:
 RTTQueries = 1,
 RTTQueue = 2,
 RTTBoth = 3

Reply:
Changed